DigitalLearningSolutions.Data/DataServices/UserDataService/AdminUserDataService.cs

```diff
@@ -332,7 +332,50 @@ public IEnumerable<AdminAccount> GetAdminAccountsByUserId(int userId)
 		                                u.ID, u.PrimaryEmail, u.FirstName, u.LastName, u.Active, u.FailedLoginCount,
 		                                c.CentreID, c.CentreName,
 		                                ucd.ID, ucd.Email, ucd.EmailVerified, ucd.CentreID,
-                                        (SELECT COUNT(*) FROM AdminSessions WHERE AdminID = aa.ID) AS AdminSessions
+                         (SELECT count(*)

```

Comment:
Is this query performant, @manishagarwalnhs? It's hitting a lot of tables and we have thousands of admin records that we are doing this for. Worried that this might be excessive. There are lots of tables included that don't need to be. The admin cannot have UpdatedByAdminID or CreatedByAdminUserID or AddedByAdminUserID if there are no AdminSessions. They also can't have commented on a ticket.

AdminSessions, SupervisorDelegates, FrameworkCollaborators

Reply:
It is completed.